proto/chart/v1/stream.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+option go_package = "chart/stream/v1;charts";
+
+package chart.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message StreamChartsAndDimensions {
+  // claim_id, node_id pair is used to identify the Node Instance
+  string claim_id = 1;
+  string node_id = 2;
+
+  // seq_id last verified sequence sent by the agent
+  int64 seq_id = 3;
+  int64 prev_seq_id = 4;
```

Comment:
I struggle to remember, Why we need `prev_seq_id` in this message?

Reply:
Hey @lvrach I am also struggling to fully remember, but I think this part of the flow helps a bit.

![Screenshot 2021-04-16 at 11 33 04 AM](https://user-images.githubusercontent.com/8328670/114996459-856a5f00-9ea7-11eb-9096-890f7f0cb4e6.png)

[_left dashed line is the charts service and the right one is the cloud's db_]

So as it looks from the docs, when we receive a new message from the agent, for a specific Node Instance, we expect that this _Agent message will have a `prev_seq_id`_ that should match the `last_seq_id` that is stored to the Cloud db.

So I would go ahead and say that actually there seems to be no reason to keep `prev_seq_id` in our db.

@chrikar what do you think? Are we missing something?

Reply:
👋🏼 Hey guys,
If the `sequence_ids` are indeed incremental numbers (which is the current agreement afaik), then indeed we don't really need this. This would make more sense to me if instead of numbers we had hashes instead. We had better confirm this with the agent devs @stelfrag @underhood  as the agent will set and also temporarily store this value.

Reply:
The agent will generate sequence ids for each message.
The de duplication process maybe introduce gaps, so if you have received No 35 and the agent has generated 36 and before it sends it there is a new message 37 that replaces 36, you will receive number 37.  So the agent should send this as prev=35, current 37. You have received 35 so there is no gap in this case. 
If you would get only 37, you wouldn't know if 36 was lost. 

(of course if the user deletes everything, we will start from No 1)

Reply:
Correct thanks a lot Stelios for chipping in. 
So it seems that the Cloud does **not** need to:

1.  store the `previous_seq_id` value to its db
2. send the `previous_seq_id` over to the Agent as part of `StreamChartsAndDimensions` command's payload.

This field will exist in the payload of every `ChartAndDimensionsUpdated` event sent by the Agent, and the Cloud will cross check it with its stored `seq_id` in order to detect gaps, if any.

Reply:
Removed the `previous_seq_id` field from proto definition.

Reply:
@stelfrag Would you agree on merging this proto definition to master?